Devscord.DiscordFramework/Workflow.cs

```diff
@@ -179,5 +188,45 @@ private async Task CallServerAddedBot(SocketGuild guild)
 
             this.OnDiscordServerAddedBot.ForEach(x => x.Invoke(discordServer));
         }
+
+        private async Task CallChannelCreated(SocketChannel socketChannel)
+        {
+            var channelFactory = this._context.Resolve<ChannelContextFactory>();
+            var channel = channelFactory.Create(socketChannel);
```

Comment:
`this._context.Resolve<ChannelContextFactory>().Create(socketChannel)`

w ten sposób zaoszczędzisz połowe linii